database/migrations/2023_02_24_173826_set_scrap_status_column_type_to_string_on_search_stats_table.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+use App\Models\SearchStat;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('search_stats', function (Blueprint $table) {
+            $table->string('scrap_status', 30)->default(SearchStat::SCRAP_STATUS_INITIALIZED)->change();
```

Comment:
How about using an ENUM column type instead of a string?

Reply:
Yeah. Enums would be probably be a better Idea.